src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +183,10 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface ArtifactDependency {
+    val disabled: Boolean
+    val sourceBuildConfiguration: BuildConfiguration
+
+    fun fetchProperties(): List<Parameter>
```

Comment:
What kind of parameters artifact dependency may have? Why not expose them as separate named properties?